stdlib/public/Concurrency/Task.swift

```diff
@@ -641,8 +641,10 @@ public func _runChildTask<T>(
   flags.isChildTask = true
 
   // Create the asynchronous task future.
-  let (task, _) = Builtin.createAsyncTaskFuture(
-      flags.bits, operation)
+  let (task, _) = Builtin.createAsyncTaskFuture(flags.bits, operation)
+
+  // TODO: move this into the creation or make part of "initialize"?
+  _taskAttachChild(child: task)
```

Comment:
Or should it become part of creating the async task...?

This is mirroring how task group does things today -- but its ABI isn't the best with many functions exposed.
Let me know if this is good or should move somewhere.

Reply:
I see, so that's not an ABI concern if a function effectively internal then? Okey will remember, though I suspect final ABI might be slightly different here hmmm...